### ISSUE TITLE
note about docker.io versus docker (older versions)

### ISSUE THREAD

**Issue by cglewis** [dockerana/dockerana#29]
should be added to readme, or move those steps out to a script, and have it check both cases

**Comment by grigio** [comment]
yes, many paths has changed with docker 1.2

**Comment by schvin** [comment]
thanks, yeah, need to tidy that up..

**Comment by grigio** [comment]
@schvin tnx, could you just say what it `/etc/init/docker.io.conf` and what should be in to make dockerama work?

I've `/etc/init/docker.conf` and `/etc/default/docker` with "Docker version 1.2.0, build fa7b24f"

**Comment by schvin** [comment]
they should work with the defaults. the reason for this ticket was to note that during installation on ubuntu, it could be called docker.io rather than docker. if you have the package installed, you should be good to go. what specific issues are you running into?

**Comment by grigio** [comment]
Yes it works fine, but some metrics aren't exposed in the dashboard like "network usage rx tx" and per container stats

**Comment by schvin** [comment]
hmm, interesting. will mess with it. thanks for the report!
